lib/widgets.js

```diff
@@ -56,9 +56,15 @@ var isSelected = function (value, choice) {
 var renderChoices = function (choices, renderer) {
     return reduce(choices, function (partialRendered, choice) {
         var isNested = is.array(choice[1]);
-        var renderData = isNested ?
-            { isNested: true, label: choice[0], choices: choice[1] } :
-            { isNested: false, value: choice[0], label: choice[1] };
+        if (choice.length == 3) {
```

Comment:
i'm not sure we need this branching - we can just set `disabled` to `false` unless a third truthy value is provided.